napari/util/app.py

```diff
@@ -0,0 +1,10 @@
+import sys
+from contextlib import contextmanager
+from PyQt5.QtWidgets import QApplication
+
+
+@contextmanager
+def app_context():
+    app = QApplication(sys.argv)
```

Comment:
This should be
```suggestion
    app = QApplication.instance() or QApplication(sys.argv)
```
since `QApplication` is a singleton.